### PR TITLE
Bluetooth: Mesh: Fix sysbuild configuration for mesh samples

### PR DESCRIPTION
**Pull request by PavelVPV**
This PR enables the extended advertising API support and the necessary amount of advertising sets in the Bluetooth controller on the network core required by the Bluetooth Mesh when a sample is build with
`--sysbuild` option.

This PR also describes which options must be enabled on the network core for the bluetooth controller to work with mesh.

### CODE REVIEW

**Comment by PavelVPV** [comment]
> LGTM. Shouldn't we remove `child image` support?

From what I understand it should stays while we support child image.

**Review by alxelax — APPROVED** [review]

File: doc/nrf/protocols/bt/bt_mesh/configuring.rst [hunk 1]
```diff
@@ -13,6 +13,11 @@ The Bluetooth® mesh support is controlled by :kconfig:option:`CONFIG_BT_MESH`,
 * :kconfig:option:`CONFIG_BT_OBSERVER` - Enables the Bluetooth Observer role.
 * :kconfig:option:`CONFIG_BT_PERIPHERAL` - Enables the Bluetooth Peripheral role.
 
+When the Bluetooth controller is located on a separate image (like in :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`), the following configuration must be applied to the Bluetooth controller configuration:
```
Comment (mia-ko) [inline doc/nrf/protocols/bt/bt_mesh/configuring.rst:16]:
```suggestion
When the Bluetooth LE Controller is located on a separate image (like on the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards), the following configuration must be applied to the Bluetooth LE Controller configuration:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
     * An issue where the shell interface was not accessible over UART because UART was used as a transport for MCUmgr SMP protocol.
       Shell is now accessible over RTT.
 
+* :ref:`bluetooth_mesh_light` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:402]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
...
+
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:408]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
...
+
+* :ref:`bluetooth_mesh_light_dim` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:414]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
...
+
+* :ref:`bluetooth_mesh_light_switch` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:420]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 6]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
...
+
+* :ref:`bluetooth_mesh_sensor_server` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:426]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 7]
```diff
@@ -394,6 +395,42 @@ Bluetooth Mesh samples
...
+
+* :ref:`bluetooth_mesh_silvair_enocean` sample:
+
+  * Fixed:
+
+    * Building with :ref:`zephyr:sysbuild` for :ref:`zephyr:nrf5340dk_nrf5340`.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:432]:
```suggestion
    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` boards.
```

**Review by mia-ko — CHANGES_REQUESTED** [review]

**Review by mia-ko — APPROVED** [review]

**Comment by jfischer-no** [comment]
@PavelVPV Please rebase

**Review by ludvigsj — APPROVED** [review]

**Comment by PavelVPV** [comment]
> @PavelVPV Please rebase

Done

**Pull request merged**